### DISCUSSION TITLE
Lesson 6 Ubuntu error

### DISCUSSION THREAD

**Discussion by Darshposhiya29** [Cyfrin/foundry-full-course-cu#182] [Q&A] [answered]
[error 2147942402 (0x80070002) when launching `ubuntu.exe']

I am getting this error while selecting ubuntu from dropdown in terminal.

**Comment by cromewar** [comment, marked as answer]
Hello @Darshposhiya29 
Please check this out: https://askubuntu.com/questions/1421805/error-2147942402-0x80070002-when-launching-ubuntu-18-04-in-wsl
and this: https://www.prowebtips.com/error-2147942402-0x80070002-when-launching-ubuntu2204-exe/?expand_article=1

**Reply by Darshposhiya29** [reply]
Thank You for your It worked for me!!!
but I have one doubt:
It leaves me with default directory : /mnt/c/WINDOWS/system32$
Is it possible to change it to just : /$

https://www.prowebtips.com/error-2147942402-0x80070002-when-launching-ubuntu2204-exe/?expand_article=1

This is the link

**Reply by Darshposhiya29** [reply]
Sorry change it to just :   :~$
